src/Microsoft.AspNet.Mvc.ViewFeatures/ViewComponents/ViewViewComponentResult.cs

```diff
@@ -76,18 +77,21 @@ public void Execute(ViewComponentContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            var viewEngine = ViewEngine ?? ResolveViewEngine(context);
```

Comment:
I'll revert. I must have sorted the `view*` rows alphabetically because reasons but ignored `isNullOrEmptyViewName` because other reasons.